compileFile.js

```diff
@@ -2,34 +2,40 @@ const { spawn } = require("child_process");
 const preferences = require("./preferencesHelper");
 const path = require("path");
 const locationHelper = require("./locationHelper");
+const config = require("./config")
+
+/**
+ * Get flags which needed for compile based on language
+ * @param {string} language programming language
+ * @param {string} filePath complete path to .c, .cpp or .py file
+ */
+function getFlags(language, filePath) {
+    const ext = config.extensions[language];
+    let flags = preferences().get("compilationFlags" + ext).split(" ");
+    if (flags[0] === "")
+        flags = [];
+
+    if (language === 'Python')
+        flags = ['-m', 'compileall', '-b', filePath, ...flags];
```

Comment:
Can you make it so that compiled bytecode, ie the `.pyc` file, is at `outputLocation` instead of the working directory?

Reply:
I wanted to do it myself at first but I couldn't find any way to that except to move it to `outputLocation` after the creation. What's your idea?